app/app.py

```diff
@@ -18,9 +28,22 @@ def get_arg_currency():
     return values
 
 
+def insert_database(jsonparam):
+    try:
+        cur = mysql.connection.cursor()
+        cur.execute("INSERT INTO dollar_libre.dollar (json) VALUES (%s)", [jsonparam])
+        mysql.connection.commit()
+        return jsonparam
+    except Exception as e:
+        return {'Status': 'Error', 'Detail': str(e)}
+
+
 @app.route('/api/v1/argcurrency', methods=['GET'])
 def arg_currency():
-    return json.dumps(get_arg_currency())
+    jsonCreated = get_arg_currency()
```

Comment:
Remove this logic from here and place it in the get_arg_currency so that:
When we fetch the values... we automatically insert it in the DB

So in the end, what arg_currency will do is just:

json = get_arg_currency()
return json.dumps(json)